examples/react-router-prototyping/webpack.skpm.config.js

```diff
@@ -0,0 +1,19 @@
+/* eslint-disable no-param-reassign */
+/* eslint-disable func-names */
+
+/**
+ * Function that mutates original webpack config.
+ * Supports asynchronous changes when promise is returned.
+ *
+ * @param {object} config - original webpack config.
+ * @param {boolean} isPluginCommand - wether the config is for a plugin command or a resource
+ */
+module.exports = function (config) {
+  /* you can change config here */
+
+  config.resolve = {
+    ...config.resolve,
+    extensions: [...config.resolve.extensions, '.jsx'],
```

Comment:
let's just use `.js` instead of `.jsx` like the other examples

Reply:
I've removed custom webpack config. Is it worth adding a note in the docs that this step is required for `.jsx` files, or is this something that should be addressed in `skpm`?

Reply:
we can easily add it to skpm yes.